src/test/java/htsjdk/samtools/ValidateSamFileTest.java

```diff
@@ -64,6 +65,18 @@ public void testValidSamFile() throws Exception {
     }
 
     @Test
+    public void testValidCRAMFileWithoutSeqDict() throws Exception {
+        final SamReader samReader = SamReaderFactory.
+                makeDefault().
+                validationStringency(ValidationStringency.SILENT).
+                referenceSequence(new File(TEST_DATA_DIR, "nm_tag_validation.fa")).
+                open(new File(TEST_DATA_DIR, "nm_tag_validation.cram"));
+        final ReferenceSequenceFile reference = new FastaSequenceFile(new File(TEST_DATA_DIR, "nm_tag_validation.fa"), true);
```

Comment:
@vadimzalunin I can not understand why do we need this check: this test will always fail since cram reader can't work with reference without index. Could you please clarify your comment?